tutorials/rendering/renderers.rst

```diff
@@ -66,16 +66,16 @@ Choose **Forward+** if:
 
 Choose **Mobile** if:
 
-    - You are developing for newer mobile devices, XR, or desktop.
+    - You are developing for newer mobile devices, desktop XR, or desktop.
     - You have relatively new hardware which supports Vulkan, Direct3D 12, or Metal.
     - You are developing a 3D game.
     - You want to use advanced rendering features, subject to the limitations
       of mobile hardware.
 
 Choose **Compatibility** if:
 
-    - You are developing for older mobile devices, or older desktop devices. The
-      Compatibility renderer supports the widest range of hardware.
+    - You are developing for older mobile devices, older desktop devices, or
+      stand-alone XR. The Compatibility renderer supports the widest range of hardware.
```

Comment:
```suggestion
      standalone XR. The Compatibility renderer supports the widest range of hardware.
```
Consider "standalone" as one word instead of hyphenated. It's currently split within the docs and "standalone" seems common in general usage